docs/casda/casda.rst

```diff
@@ -29,11 +29,11 @@ For example:
     obs_publisher_did       s_ra           s_dec           obs_release_date
                             deg             deg
     ----------------- --------------- ---------------- ------------------------
-           cube-11460 326.52250211117 -43.891190432671 2020-11-18T09:49:27.134Z
-           cube-11461 326.52250211117 -43.891190432671 2020-11-18T09:49:27.134Z
-           cube-11462 326.52250211117 -43.891190432671 2020-11-18T09:49:27.134Z
-           cube-11463 326.52250211117 -43.891190432671 2020-11-18T09:49:27.134Z
-           cube-11464 326.52250211117 -43.891190432671 2020-11-18T09:49:27.134Z
+            cube-1170 333.70448386919 -45.966341151806 2019-01-30T13:00:00.000Z
```

Comment:
is the extra leading space here intentional?

Reply:
(I would be inclined to switch to IGNORE_OUTPUT if this would need an update in the near future again, but for now just changed the expected output)